tests/unit/core/test_deploy_export.py

```diff
@@ -53,28 +54,52 @@
     # This import causes pycuda to automatically manage CUDA context creation and cleanup.
     import pycuda.driver as cuda
 
-    sys.path.append(os.path.join(path.dirname(path.abspath(__file__)), 'trt_ONNX'))
-    import tensorrt_loaders as loaders
-    import tensorrt_runner as runner
+    from .tensorrt_loaders import (
+        DefaultDataLoader,
+        DataLoaderCache,
+        OnnxFileLoader,
+        OnnxNetworkLoader,
+        BuildEngineLoader,
+    )
+    from .tensorrt_runner import TensorRTRunnerV2
 except:
     # Skip tests.
     pytestmark = pytest.mark.skip
 
 
 @pytest.mark.usefixtures("neural_factory")
 class TestDeployExport(TestCase):
+    def setUp(self):
```

Comment:
Hey, why you added this once again?

The "neural_factory" fixture that the class uses is creating the Neural Factory (self.nf) already. It also handles the device/placement...

Besides, the "run_only_on("GPU") pytest marker already makes sure that a given test will run only on GPU. So those lines are redundant...

Also, not sure about the logging level. When you use pytest, it captures the streams and not print anything unless you tell it specifically. So once again, setting logger level to WARNING (which will result in skipping INFO) will have no impact on that is shown.

In short, the whole setUp() seems to be redundant, please considering removing  it.